### ISSUE TITLE
After received SIGHUP, log message is printed twice when --log_config is not specified

### ISSUE THREAD

**Issue by rimms** [jubatus/jubatus#962]
After received SIGHUP, log message is printed twice when ``--log_config`` is **not** specified.

1. Start up Server without ``--log_config`` option.

  ```
  $ jubaclassifier -f arow.json &
  $ echo $!
  19835
  ```
2. Send SIGHUP

  ```
  $ kill -s SIGHUP 19835
  2015-03-23 21:30:17,981 19836 INFO  [signals.cpp:118] caught SIGHUP
 ```

3. Send SIGHUP again, log message was printed twice.

  ```
  $ kill -s SIGHUP 19835
  2015-03-23 21:31:05,586 19836 INFO  [signals.cpp:118] caught SIGHUP
  2015-03-23 21:31:05,586 19836 INFO  [signals.cpp:118] caught SIGHUP <==
 ```

4. Send SIGHUP again, log message was printed 3 times.

 ```
 $ kill -s SIGHUP 19835
 2015-03-23 21:31:18,414 19836 INFO  [signals.cpp:118] caught SIGHUP
 2015-03-23 21:31:18,414 19836 INFO  [signals.cpp:118] caught SIGHUP <==
 2015-03-23 21:31:18,414 19836 INFO  [signals.cpp:118] caught SIGHUP <== 
 ```

This situation will not occur  when ``--log_config`` is specified.

Note: I found this in test of #896 patch.

**Comment by kmaehashi** [comment]
I noticed that calling `configure()` does not remove existing appenders.  So ConsoleAppender is created every time SIGHUP is called, causing the same log message to appear multiple times.

https://github.com/jubatus/jubatus/blob/0.8.6/jubatus/server/common/logger/logger.cpp#L95

```
log4cxx::LayoutPtr layout(
    new log4cxx::PatternLayout("%d %X{tid} %-5p [%F:%L] %m%n"));
log4cxx::AppenderPtr appender(new log4cxx::ConsoleAppender(layout));
log4cxx::BasicConfigurator::configure()
```

https://github.com/jubatus/jubatus/blob/0.8.6/jubatus/server/framework/server_util.cpp#L68

```
void configure_logger(const std::string& log_config) {
  if (log_config.empty()) {
    common::logger::configure();
  } else {
    // ... snip ...
  }
  // ... snip ...
  common::set_action_on_hup(jubatus::util::lang::bind(
      configure_logger, jubatus::util::lang::ref(log_config)));
}
```

We don't have to re-configure log4cxx when log_config is not specified.

**Comment by kmaehashi** [comment]
Fixed via #1084, #1086
